### PR TITLE
Please add APC SmartUPS SMT1000RMI2UC

### PR DESCRIPTION
**Pull request by sthierolf**
Please add APC SmartUPS SMT1000RMI2UC. This model uses European/German C14/C13 sockets.

Datasheet: [APC Smart-UPS, Line Interactive, 1000VA, Rackmontage 2HE, 230V, 4 IEC C13-Stecker, SmartConnect Port+SmartSlot, AVR, LCD](https://www.apc.com/de/de/product/SMT1000RMI2UC/apc-smartups-line-interactive-1000va-rackmontage-2he-230v-4-iec-c13stecker-smartconnect-port+smartslot-avr-lcd/)

### CODE REVIEW

File: device-types/APC/SMT1000RMI2UC.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+manufacturer: APC
+model: Smart-UPS 1000VA, Rack Mount, LCD 230V with SmartConnect Port
+slug: smart-ups-1000va-rack-mount-lcd-230-with-smartconnect-port
+part_number: SMT1000RMI2UC
+u_height: 2
+is_full_depth: true
```
Comment (danner26) [inline device-types/APC/SMT1000RMI2UC.yaml:7]: Please add weight if you can

Comment (danner26) [inline device-types/APC/SMT1000RMI2UC.yaml:7]: Please add airflow if you can

File: device-types/APC/SMT1000RMI2UC.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+manufacturer: APC
+model: Smart-UPS 1000VA, Rack Mount, LCD 230V with SmartConnect Port
```
Comment (danner26) [inline device-types/APC/SMT1000RMI2UC.yaml:3]: This is not the correct model name, realistically this is too long to be correct. It looks like `SMT1000RMI2UC` would be a better option

File: device-types/APC/SMT1000RMI2UC.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+slug: smart-ups-1000va-rack-mount-lcd-230-with-smartconnect-port
```
Comment (danner26) [inline device-types/APC/SMT1000RMI2UC.yaml:4]: Please append `apc-` to the slug. I suggest you change this to `apc-smt1000rmi2uc`

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/APC/SMT1000RMI2UC.yaml (hunk 1 above)
Comment (sthierolf) [reply device-types/APC/SMT1000RMI2UC.yaml:10]: added weight/weight_unit. Checked if float (22.5) is accepted.
As requested, added prefix apc- and airflow. Technically there is not much of airflow, the UPS just has a fan on its power supply on the rear.

**Review by sthierolf — COMMENTED** [review]
added fields as requested

File: device-types/APC/SMT1000RMI2UC.yaml (hunk 2 above)
Comment (danner26) [reply device-types/APC/SMT1000RMI2UC.yaml:3]: @sthierolf please check this comment as well

**Review by danner26 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**